### PR TITLE
clue scroll plugin: add support for Falo the Bard clues

### LINKED ISSUE #2730: Add items needed to show Falo the Bard to the Clue Scroll Plugin

### PR DESCRIPTION
**Pull request by Twiglet1022**
Closes #2730 

I only just realised these weren't supported yet when I got on my master clue so I added them in since I had one to test on.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+import net.runelite.client.ui.overlay.components.LineComponent;
+import net.runelite.client.ui.overlay.components.PanelComponent;
+import net.runelite.client.ui.overlay.components.TitleComponent;
+
+@Getter
+public class FaloTheBardClue extends ClueScroll implements TextClueScroll, NpcClueScroll
+{
+	private static final Set<FaloTheBardClue> CLUES = ImmutableSet.of(
+		new FaloTheBardClue("A blood red weapon, a strong curved sword, found on the island of primate lords.", item(DRAGON_SCIMITAR)),
+		// I'm not sure how one would detect a full god book but the text clarifies it
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java]: Don't the full books have their own ids?

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+		this.text = text;
+		this.itemRequirements = itemRequirements;
+	}
+
+	@Override
+	public void makeOverlayHint(PanelComponent panelComponent, ClueScrollPlugin plugin)
+	{
+		panelComponent.getChildren().add(TitleComponent.builder().text("Falo the Bard Clue").build());
+
+		{
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java]: ?

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+	{
+		return range(null, startItemId, endItemId);
+	}
+
+	private static RangeItemRequirement range(String name, int startItemId, int endItemId)
+	{
+		return new RangeItemRequirement(name, startItemId, endItemId);
+	}
+
+	private static final WorldPoint LOCATION = new WorldPoint(2689, 3550, 0);
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java]: We generally put static fields at the top of classes

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java (hunk 1 above)
Comment (Twiglet1022) [reply runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/FaloTheBardClue.java]: Yes you're right, for some reason I thought partially filled books had the same ID. Instead they're called damaged books until they're filled.

**Pull request merged**